js_src/rdm.js

```diff
@@ -340,20 +340,26 @@ angular.module('rdmApp', [])
      var token = tokens[j];
      var hex_suffix_match = token.match(/^([\da-fA-F]{1,2})h$/);
      if (hex_suffix_match) {
-      binary_data.push(parseInt(hex_suffix_match[0], 16));
```

Comment:
Can we have a comment at the top of this block reminding people of the broken way capture groups work in JS (it gets even messier if it's a global match with groups!